Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: I can see the generic Content Type icon attached to the Content item name
```

Comment:
I'd skip the "can see": what about `A generic Content Type icon is attached to the Content item's name" ?